src/test/unit/symbol-store-db.test.js

```diff
@@ -5,6 +5,9 @@
 
 import SymbolStoreDB from '../../profile-logic/symbol-store-db';
 import { completeSymbolTableAsTuple } from '../fixtures/example-symbol-table';
+// fake-indexeddb no longer includes a structuredClone polyfill, so we need to
+// import it explicitly.
+import 'core-js/stable/structured-clone';
 import { indexedDB, IDBKeyRange } from 'fake-indexeddb';
```

Comment:
Should we use `autoMockIndexedDB` here and in the tests below instead of defining the polyfill manually?